src/editors/data/redux/thunkActions/problem.js

```diff
@@ -3,14 +3,10 @@ import { actions } from '..';
 import { OLXParser } from '../../../containers/ProblemEditor/data/OLXParser';
 import { parseSettings } from '../../../containers/ProblemEditor/data/SettingsParser';
 import { ProblemTypeKeys } from '../../constants/problem';
-import ReactStateOLXParser from '../../../containers/ProblemEditor/data/ReactStateOLXParser';
 import { blankProblemOLX } from '../../../containers/ProblemEditor/data/mockData/olxTestData';
 
-export const switchToAdvancedEditor = () => (dispatch, getState) => {
```

Comment:
This change is related to my changes because when you switch from the visual to advanced editor it is parsing the current content. Earlier we determined that the rawOLX was not updating so I thought it was better to remove the code then add a new line that doesn't really contribute anything.

Reply:
However, after looking back at the code again, I figured out why the rawOLX was not updating when switching from the visual editor to the advanced editor. The variable that we were calling in `updateField` was `rawOlx`; however, the variable's name in the redux store is `rawOLX`. 🤦‍♀️

Reply:
ugh that's brutal glad you caught that!